lib/membrane_http_adaptive_stream/sink_bin.ex

```diff
@@ -100,6 +103,20 @@ defmodule Membrane.HTTPAdaptiveStream.SinkBin do
         Name that will be used to name the media playlist for the given track, as well as its header and segments files.
         It must not contain any URI reserved characters
         """
+      ],
+      muxer_segment_duration_range: [
+        spec: CMAF.SegmentDurationRange.t(),
+        description: """
+        The target duration range of regular segments produced by the muxer.
+        """
+      ],
+      muxer_partial_segment_duration_range: [
```

Comment:
Aren't these used by both muxer and sink? Do we need the muxer_ prefix?

Reply:
Probably not, will change that. It was like this before so I kept it that way.